### PR TITLE
feat(optimus): [RND-65933] Add standalone link

### PR DESCRIPTION
**Pull request by Gladkowski**
#### Summary

- add standalone link
- `inherit` and `external` properties are not implemented

#### Testing steps

*Info about test cases. If you think the issue is not testable, describe why.*

#### Follow-up issues

*Is there some action/cleanup needed after this PR is merged or deployed (e.g. consolidate some part outside the scope of this PR, etc)? Create issues for it with deadline and note them here and in the PR comments.*

#### Check during review

- Verify against YouTrack issue.
- Is the PR over 300 additions? Consider rejecting it with advice to split it. Is it over 500 additions? It should definitely be rejected.
- Unused code removed.
- Build passing.
- Is it a bug fix? Check that it is covered by a proper test (unit or integration).

### CODE REVIEW

File: optimus/lib/src/standalone_link.dart [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+          onEnter: (_) => _onHoverChanged(true),
+          onExit: (_) => _onHoverChanged(false),
+          child: GestureDetector(
+            onTap: widget.onPressed,
+            onTapDown: (_) => setState(() => _isTappedDown = true),
+            onTapUp: (_) => setState(() => _isTappedDown = false),
+            onTapCancel: () => setState(() => _isTappedDown = false),
+            child: Text.rich(
+              TextSpan(
+                children: [
```
Comment (chaeMil) [inline optimus/lib/src/standalone_link.dart:64]: Why is this here? It looks and function without it.

**Review by chaeMil — CHANGES_REQUESTED** [review]

**Review by chaeMil — APPROVED** [review]

File: optimus/lib/src/standalone_link.dart [hunk 2]
```diff
@@ -28,11 +28,13 @@ class OptimusStandaloneLink extends StatefulWidget {
     this.onPressed,
     required this.text,
     this.size = OptimusLinkSize.normal,
+    this.isInherit = false,
   }) : super(key: key);
 
   final VoidCallback? onPressed;
   final String text;
   final OptimusLinkSize size;
+  final bool isInherit;
```
Comment (ookami-kb) [inline optimus/lib/src/standalone_link.dart:37]: When using like this, it's even more confusing than before. How should user understand that `isInherit` makes the link color black?

Can we somehow make it aligned with optimus guidelines and create a more meaningful parameter name?

Comment (Gladkowski) [reply optimus/lib/src/standalone_link.dart:37]: What about `isDefault`?

File: optimus/lib/src/standalone_link.dart [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+  }
+
+  @override
+  Widget build(BuildContext context) => OptimusEnabled(
+        isEnabled: widget.onPressed != null,
+        child: MouseRegion(
+          onEnter: (_) => _onHoverChanged(true),
+          onExit: (_) => _onHoverChanged(false),
+          child: GestureDetector(
+            onTap: widget.onPressed,
```
Comment (ookami-kb) [inline optimus/lib/src/standalone_link.dart:70]: Why is `onTap` used both here and in `_gestureRecognizer`?

File: optimus/lib/src/standalone_link.dart [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+            onTapDown: (_) => setState(() => _isTappedDown = true),
+            onTapUp: (_) => setState(() => _isTappedDown = false),
+            onTapCancel: () => setState(() => _isTappedDown = false),
+            child: Text.rich(
```
Comment (ookami-kb) [inline optimus/lib/src/standalone_link.dart:72]: Actually, I think we don't need rich text at all for a standalone link.

Comment (Gladkowski) [reply optimus/lib/src/standalone_link.dart:72]: well, valid idea

**Review by ookami-kb — APPROVED** [review]

**Pull request merged**